test/gfx/test.sh

```diff
@@ -7,7 +7,7 @@ errtmp="$(mktemp)"
 
 # Immediate expansion is the desired behavior.
 # shellcheck disable=SC2064
-trap "rm -f ${errtmp@Q}" EXIT
+trap "rm -f ${errtmp@Q} result.png result.2bpp result.tilemap result.attrmap result.pal" EXIT
```

Comment:
Might as well

```suggestion
trap "rm -f ${errtmp@Q} result.*" EXIT
```

Reply:
I'd prefer to be explicit here, just like we are in the .gitignore. If I have my own result.log file there for some reason I don't want it deleted.

Reply:
Okay, fair. We should still add `.palmap` to the list, though.